xmpp.go

```diff
@@ -561,11 +562,13 @@ func (c *Client) startTLSIfRequired(f *streamFeatures, o *Options, domain string
 func (c *Client) startStream(o *Options, domain string) (*streamFeatures, error) {
 	if o.Debug {
 		c.p = xml.NewDecoder(tee{c.conn, DebugWriter})
+		StanzaWriter = io.MultiWriter(c.conn, DebugWriter)
 	} else {
 		c.p = xml.NewDecoder(c.conn)
+		StanzaWriter = c.conn
```

Comment:
This change appears unsafe in the presence of multiple clients. All client instances will assign and write to the same package scope `StanzaWriter`.

@mattn Can this be reverted?

Reply:
@mdosch Can you handle this?